components/ModFacilitySubmissionLeftNavbar.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+    <div class="flex flex-col">
+        <h1 class="font-bold text-lg p-1 mb-2 text-ellipsis">
+            {{ moderationSubmissionsStore.selectedSubmissionData?.facility?.nameEn
+                || $t("modPanelFacilitySubmissionLeftNavbar.facilityNameUnknown") }}
+        </h1>
+        <div class="flex flex-col items-start">
+            <button
+                data-testid="submission-form-leftNav-contactInformation"
+                :class="{ 'bg-lightBlue': activeSection === 'contact-information',
+                          'bg-transparent': activeSection !== 'contact-information' }"
+                class="w-full py-4 my-2 text-sm text-start pl-2 rounded border-b-2 border-slate-200"
+                @click="scrollToSectionOfFacilityForm('contact-information')"
+            >
+                {{ $t("modPanelFacilitySubmissionLeftNavbar.contactInformation") }}
+            </button>
+            <button
+                data-testid="submission-form-leftNav-addresses"
+                :class="{ 'bg-lightBlue': activeSection === 'addresses', 'bg-transparent': activeSection !== 'addresses' }"
```

Comment:
let's make these strings into an enum. 
this will help prevent spelling errors and future devs to follow along better